### PR TITLE
[KOB-3556] - Support Jasmine framework

### PR DESCRIPTION
**Pull request by anvuong87**
automation web, app tests on Kobiton's devices using jasmine

### CODE REVIEW

File: javascript/spec/jasmine_app_spec.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import 'babel-polyfill';
+import 'colors';
+import wd from 'wd';
+
+let driver;
+const kobitonServerConfig = {
+  protocol: 'https',
+  host: 'api.kobiton.com',
```
Comment (khanhdodang) [inline javascript/spec/jasmine_app_spec.js]: Get value from global variable

File: javascript/README.md [hunk 2]
```diff
@@ -93,6 +93,8 @@ npm run android-app-test
 npm run ios-web-test
 npm run ios-app-test
 npm run multiple-devices-test
+npm run jasmine_web_test
```
Comment (khanhdodang) [inline javascript/README.md]: change `_` to `-`

File: javascript/spec/jasmine_app_spec.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      }
+      catch (err) {
+        console.error(`Failed to quit driver: ${err}`);
+      }
+    }
+  });
+
+  it('should get correct category name', async () => {
+    await driver.elementByClassName("android.widget.TextView")
+     .text().then(function(text) {
```
Comment (khanhdodang) [inline javascript/spec/jasmine_app_spec.js]: try to remove `function`

File: javascript/README.md (hunk 2 above)
Comment (anvuong87) [reply javascript/README.md]: fixed

File: javascript/spec/jasmine_app_spec.js (hunk 1 above)
Comment (anvuong87) [reply javascript/spec/jasmine_app_spec.js]: fixed

File: javascript/spec/jasmine-web-spec.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+const apiKey = process.env.KOBITON_API_KEY
+const deviceName = process.env.KOBITON_DEVICE_NAME || '*'
+
+let driver;
+const kobitonServerConfig = {
+  protocol: 'https',
+  host: 'api.kobiton.com',
+  auth: `${username}:${apiKey}`
+};
+const desiredCapabilities = {
```
Comment (khanhdodang) [inline javascript/spec/jasmine-web-spec.js]: It just runs on Android devices

Comment (anvuong87) [reply javascript/spec/jasmine-web-spec.js]: files renamed, currently will support for android device (web/app), we'll update for iOS later.

File: javascript/spec/jasmine-android-web-spec.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+      }
+      catch (err) {
+        console.error(`Failed to quit driver: ${err}`);
+      }
+    }
+  });
+
+  it('should return the title that contains Kobiton', async () => {
+    await driver.get('https://www.google.com')
+    .waitForElementByName('q')
```
Comment (khanhdodang) [inline javascript/spec/jasmine-android-web-spec.js]: check eslint

Comment (anvuong87) [reply javascript/spec/jasmine-android-web-spec.js]: fixed

**Pull request merged**